index.html

```diff
@@ -11,5 +11,6 @@
 <body>
     <div id="root"></div>
     <script src="/config.js"></script>
+    <script src="http://localhost:3002/plugin.subscription.js"></script>
```

Comment:
Why is there `localhost:3002` and where is `plugin.subscription.js` built?

Reply:
hmmm, not cool.
This should be configured somehow from config.js.

plugin.subscription.js  is just a POC plugin but we need to be able to plug-in any plugin.

Reply:
@wiadev  can u pls fix it?